src/components/Button/Button.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link Button }
+ */
+export interface ButtonProps extends BaseButtonProps {
+  buttonName?: string;
+  buttonType?: React.ButtonHTMLAttributes<HTMLButtonElement>["type"];
+  children: React.ReactNode;
+  disabled?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+}
+
+/**
+ * The button element, that can be used to help users
+ * carry out an action.
+ */
+export const Button: React.FunctionComponent<ButtonProps> = (
+  props: ButtonProps
+): JSX.Element => {
+  const {
+    id,
+    className,
+    isStartButton,
+    buttonName,
+    buttonType,
+    children,
+    disabled,
+    onClick
+  } = props;
+  return (
+    <button
+      id={id}
+      className={classNames("govuk-button", className, {
+        "govuk-button--start": isStartButton
+      })}
+      name={buttonName}
+      type={buttonType}
+      disabled
+      aria-disabled={disabled}
+      data-module="govuk-button"
+      onClick={onClick}
+    >
+      {children}
+    </button>
+  );
+};
+
+Button.propTypes = {
+  id: PropTypes.string,
+  className: PropTypes.string,
+  isStartButton: PropTypes.bool,
+  buttonName: PropTypes.string,
+  buttonType: PropTypes.oneOf(["submit", "button", "reset", undefined]),
```

Comment:
Do you need `undefined` in the list? Isn't that implicit in not adding `.isRequired`?